### ISSUE TITLE
Translate Bones Doesn't Translate Vertex Groups When Multiple Armatures Exist

### ISSUE THREAD

**Issue by Nifty255** [michaeldegroot/cats-blender-plugin#45]
Importing a single model via PMX works fine; however, importing one, translating it, importing another, and then attempting to translate it, will not translate the newly imported mesh's vertex groups.

**Comment by Darkblader24** [comment]
Did you select the armature in the list above the Fix Model button?

**Comment by Nifty255** [comment]
Yes, I did. It translated the armature, but not the associated mesh. The mesh was also a child of the armature with a modifier, as if it were just imported.

**Comment by Darkblader24** [comment]
I just tested it and had no issues. Not sure what goes wrong there

**Comment by Nifty255** [comment]
It could be because I wasn't "fixing" my subsequently imported models, since fixing usually just errors out due to things being in 20 different layers.

**Comment by Darkblader24** [comment]
Oh, it does? Never seen that before. What error appears then?
